WordPress/Classes/ViewRelated/Stats/Insights/ViewsVisitors/ViewsVisitorsLineChartCell.swift

```diff
@@ -54,14 +54,34 @@ struct StatsSegmentedControlData {
     }
 
     var differenceLabel: String {
-        let stringFormatValue = differencePercent != 0 ? "%@%@ (%@%%)" : "%@%@"
-        let stringFormat = NSLocalizedString(stringFormatValue, comment: "Difference label for Insights Overview stat, indicating change from previous period. Ex: +99.9K(5%)")
-        return String.localizedStringWithFormat(
-            stringFormat,
-            difference < 0 ? "" : "+",
-            difference.abbreviatedString(),
-            differencePercent.abbreviatedString()
-        )
+        // We want to show something like "+1.2K (5%)" if we have a percentage difference and "1.2K" if we don't.
+        // Because localized strings need to be strings literal, we cannot embed any conditional logic in the `localizedString...` call.
```

Comment:
👍  Nice catch!

Not sure how that was missed in the original PR  😓, did `gentrings` warned you about it during code freeze, telling you that the parameters have to be static values? If so, maybe we should run `genstrings` on CI on each PR (even if we don't commit the generated `Localizable.strings` on those PR CI runs, but just to gather potential warnings generated by it)? 🤔

Reply:
> but I worry if it's a waste to spin up a macOS box etc. "just" for that.

Fair point.